pkgs/applications/editors/emacs/build-support/wrapper.nix

```diff
@@ -169,6 +169,7 @@ runCommand (lib.appendToName "with-packages" emacs).name
             (load "$emacs/share/emacs/site-lisp/site-start"))
           ;; "$out/share/emacs/site-lisp" is added to load-path in wrapper.sh
           ;; "$out/share/emacs/native-lisp" is added to native-comp-eln-load-path in wrapper.sh
+          (setenv "PATH" (concat "$out/bin:" (getenv "PATH"))) ; some packages blindly overwrite exec-path with $PATH
```

Comment:
Thanks for your contribution!

Could you give me a specific example what issue "some packages blindly overwriting exec-path with $PATH" causes (ideally with a minimal reproducible example) so that we can understand the issue to be fixed better?  You mentioned Doom and envrc.  Many users, including myself, use envrc and there is no issue.